ci/ci_steps.sh

```diff
@@ -37,8 +37,11 @@ else
     cp -f "$ENVOY_BUILD_DIR"/envoy/source/exe/envoy ./build_release
     # This script builds a lyft/envoy image and pushes that image on merge to master.
     ./ci/docker_push.sh
-    # This script runs on every PRs release run to test the docker examples.
-    ./ci/verify_examples.sh
+    if [ "$TRAVIS_PULL_REQUEST" == "false" ] && [ "$TRAVIS_BRANCH" == "master" ]
```

Comment:
Your teachings are definitely permeating into my brain! Before at the same time you wrote this comment, I pushed the latests commit. I am moving the verification into the docker_push file. As you said, no reason to have the guards in different places.